### PR TITLE
lib_nbgl: Update implementation for LNX and LNSP

### PR DESCRIPTION
**Pull request by xchapron-ledger**
## Description

lib_nbgl: Update implementation for LNX and LNSP

## Changes include

- [ ] Bugfix (non-breaking change that solves an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [x] Breaking change (change that is not backwards-compatible and/or changes current functionality)
- [ ] Tests
- [ ] Documentation
- [ ] Other (for changes that might not fit in any category)

## Breaking changes

Change the API, but no one was using it and it's for the best: unification with other devices.

### CODE REVIEW

File: Makefile.standard_app [hunk 1]
```diff
@@ -134,6 +134,10 @@ endif
 #####################################################################
 #                              NBGL                                 #
 #####################################################################
+ifeq ($(ENABLE_NBGL_FOR_NANOS), 1)
```
Comment (fbeutin-ledger) [inline Makefile.standard_app:137]: why is this in lnx / lns+ ?

Comment (xchapron-ledger) [reply Makefile.standard_app]: Done

**Review by cedelavergne-ledger — APPROVED** [review]

File: Makefile.standard_app [hunk 2]
```diff
@@ -79,6 +79,38 @@ ifneq ($(DISABLE_DEFAULT_IO_SEPROXY_BUFFER_SIZE), 1)
...
+
+ifeq ($(ENABLE_NBGL_QRCODE), 1)
+ifeq ($(TARGET_NAME),$(filter $(TARGET_NAME),TARGET_STAX TARGET_FLEX))
+    DEFINES += NBGL_QRCODE
+    SDK_SOURCE_PATH += qrcode
+endif
+endif
+
+ifeq ($(ENABLE_NBGL_KEYBOARD), 1)
+ifeq ($(TARGET_NAME),$(filter $(TARGET_NAME),TARGET_STAX TARGET_FLEX))
```
Comment (nroggeman-ledger) [inline Makefile.standard_app:103]: NBGL_KEYBOARD & NBGL_KEYPAD exist also on Nanos

**Review by nroggeman-ledger — DISMISSED** [review]

File: Makefile.standard_app (hunk 2 above)
Comment (xchapron-ledger) [reply Makefile.standard_app:103]: Yeap but as of now the corresponding useCase functions are not implemented.
So we can probably keep this like it for now.

Comment (xchapron-ledger) [reply Makefile.standard_app]: Fixed

**Review by nroggeman-ledger — APPROVED** [review]

**Pull request merged**